mappings/net/minecraft/entity/ai/brain/task/MemoryTransferTask.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4809 net/minecraft/entity/ai/brain/task/MemoryTransferTask
+	FIELD field_22288 runPredicate Ljava/util/function/Predicate;
+	FIELD field_22289 sourceType Lnet/minecraft/class_4140;
+	FIELD field_22290 targetType Lnet/minecraft/class_4140;
+	FIELD field_22291 durationRange Lnet/minecraft/class_4801;
```

Comment:
`durationRange` seems to be a bit redundant, `duration` seems fine to me- would like more thoughts though

Reply:
Renamed to `duration`